crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/bootstrap.rs

```diff
@@ -197,6 +198,7 @@ mod tests {
     #[tokio::test(flavor = "multi_thread")]
     #[ignore = "flaky"]
     async fn test_bootstrap() {
+        /*
```

Comment:
leftover or do we want this back in the future?

Reply:
same as above